docs/api-reference/react/deckgl.md

```diff
@@ -13,56 +13,44 @@ The `DeckGL` class is a React wrapper of the `Deck` JavaScript class which expos
 // Basic usage
 import DeckGL, {ScatterplotLayer} from 'deck.gl';
 
-const App = (viewState, data) => (
+const App = (data) => (
   <DeckGL
-    {...viewState}
+    longitude={-122.45}
```

Comment:
OK, although I'd personally would not promote this component-wise style for the react version. While I don't see any need to remove it, I think it is a bit non-optimal because once more functionality is added, it requires unpacking and spoon-feeding in the view state.